features/_your_results/period_percent_table.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  As a user who has just completed the WPCC tool, I should be able to see helpful information on how the yearly increments is done. 
+
+  Scenario: Display table of percentages for each period
+    Given I am on the YourDetailsPage
+    When I fill in my details
+    And I click the Next button
+    And I move on to the results page
+    Then I should see a table with a column heading for each period
```

Comment:
@tomas-stefano : This is just a static table, which shows the statutory percentages set by the government- the very basic how workplace contribution works. 
I think you're mixing this one with other edge / exceptional cases, where contributions/ percentages are over what is minimal / standard set by the govt. Hope it makes sense?